### ISSUE TITLE
Implement better way to differentiate methods

### ISSUE THREAD

**Issue by mattgd** [mattgd/UnitConverter#11]
For example, in conversions like the one below, method names can't really be used as effectively to convert the units.

```
# Watts to Ohms given Amps
def w_to_o(watts, amps):
    return watts / (amps * amps)

# Watts to Ohms given Volts
def w_to_o2(watts, volts):
    return (volts * volts ) / watts
```

Right now, most units are converted using the abbreviations from the user input (ex: cm to in) and then selecting the method based on those abbreviations (cm_to_in method), but would like to move to a more widely compatible method for using the API in other projects where user input isn't important.

For reference, this is how the correct method is selected.

```
conversion_method = units_from + '_to_' + units_to # Name of method to use
thismodule = sys.modules[__name__] # This module as object

try:
   value = getattr(thismodule, conversion_method)(value)
    return str(round(value, decimal_places)) + units_to
except AttributeError:
   return 'Incompatible units.'
```

**Comment by ibiBgOR** [comment]
I want to look at this one

**Comment by ibiBgOR** [comment]
My thoughts are implemented in my pull request: #26 
Hope that satisfied you a little bit
